app/controllers/import_controller.rb

```diff
@@ -217,6 +163,88 @@ def import_epi_x_field(patient, field, row, row_ind, col_num, international_addr
     end
   end
 
+  def import_sdx_field(patient, field, row, row_ind, col_num)
+    return if [nil, 'N/A'].include?(row[col_num])
+
+    if %i[travel_related_notes flight_or_vessel_carrier flight_or_vessel_number].include?(field) # fields represented by multiple columns
+      value = "#{SDX_HEADERS[col_num]}: #{import_field(field, row[col_num], row_ind)}"
+      patient[field] = patient[field].blank? ? value : "#{patient[field]}; #{value}"
+    elsif field == :exposure_notes # CreateDate
+      value = "#{SDX_HEADERS[col_num]}: #{import_field(field, Date.strptime(row[col_num], '%m%d%Y'), row_ind)}"
+      patient[field] = patient[field].blank? ? value : "#{patient[field]}; #{value}"
+    elsif field == :alternate_contact_name
+      patient[field] = patient[field].blank? ? row[col_num] : "#{patient[field]} #{row[col_num]}"
+    elsif %i[date_of_arrival date_of_departure date_of_birth].include?(field)
+      patient[field] = import_field(field, Date.strptime(row[col_num], '%m%d%Y'), row_ind)
+    elsif %i[primary_telephone secondary_telephone alternate_primary_telephone].include?(field)
+      phone = Phonelib.parse(row[col_num], 'US')
+      patient[field] = phone.full_e164 || row[col_num] && return unless phone.full_e164.blank? || phone.full_e164.sub(/^\+1+/, '').length != 10
+
+      patient[:alternate_international_telephone] = row[col_num] if field == :alternate_primary_telephone
```

Comment:
Seems to be an issue with this. In the case of a valid US phone number, it's getting assigned to both the primary and international telephone fields, when it should just be the primary.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1170/commits/ba533c7bb6cdb3e0570420e8d97a3d7a6b0230a1